.github/workflows/periodic_benchmarks.yaml

```
@@ -0,0 +1,96 @@
+# Initial Source: pybop-team/PyBop
+
+# This workflow periodically runs the benchmarks suite in benchmarks/
+# using asv and publish the results, effectively updating
+# the display website hosted in the pybop-bench repo
+
+# Steps:
+# - Benchmark all commits since the last one that was benchmarked
+# - Push results to pybop-bench repo
+# - Publish website
+name: Benchmarks
+on:
+  # Everyday at 12 pm UTC
+  schedule:
```

To help debug this, you can add:

```suggestion
name: Benchmarks
on:
  push:
  # Everyday at 12 pm UTC
  schedule:
```

and once this is done, the workflow will start showing up in the Actions tab. And then the `push:` event can be removed in a further commit and `workflow_dispatch:` will work (i.e., workflows require their files to be pushed at least once).